design-doc-template.adoc

```diff
@@ -4,125 +4,95 @@ categories:
 
 == Backwards Compatibility
 
-// Does this enhancement affect backwards compatibility with previously released
-// versions of WildFly?
-// Can the identified incompatibility be avoided?
+__<Does this enhancement affect backwards compatibility with previously released versions of WildFly? Can the identified incompatibility be avoided?>__
 
 === Default Configuration
 
+__<What is the impact of this feature in the default configuration(s) provided by WildFly?>__
+
 === Importing Existing Configuration
 
+TBD
+
 === Deployments
 
+__<What is the impact of this feature for deployments?>__
+
 === Interoperability
 
-//== Implementation Plan
-////
-Delete if not needed. The intent is if you have a complex feature which can 
-not be delivered all in one go to suggest the strategy. If your feature falls 
-into this category, please mention the Release Coordinators on the pull 
-request so they are aware.
-////
+__<Is this feature impacting interoperability?>__
 
-== Security Considerations
+== Implementation Plan
 
-////
-Identification if any security implications that may need to be considered with this feature
-or a confirmation that there are no security implications to consider.
-////
+__<This section is optional. If you have a complex feature which can not be delivered all in one go, suggest the strategy.>__
+
+== Thread Model
+
+__<What impact on security does this feature have?>__
 
 == Test Plan
 
-== Community Documentation
-////
-Generally a feature should have documentation as part of the PR to wildfly master, or as a follow up PR if the feature is in wildfly-core. In some cases though the documentation belongs more in a component, or does not need any documentation. Indicate which of these will happen.
-////
+__<How do you plan to test this feature?>__
+
+== Documentation Plan
+
+__<Describe how this feature will be documented or illustrated. Generally a feature should have documentation as part of the PR to wildfly master, or as a follow up PR if the feature is in wildfly-core. In some cases though the feature will bring additional content (such as quickstarts, guides, etc.). Indicate which of these will happen>__
+
 == Release Note Content
-////
-Draft verbiage for up to a few sentences on the feature for inclusion in the
-Release Note blog article for the release that first includes this feature. 
-Example article: http://wildfly.org/news/2018/08/30/WildFly14-Final-Released/.
-This content will be edited, so there is no need to make it perfect or discuss
-what release it appears in.  "See Overview" is acceptable if the overview is
-suitable. For simple features best covered as an item in a bullet-point list 
-of features containing a few words on each, use "Bullet point: <The few words>" 
-////
+
+__<Draft verbiage for up to a few sentences on the feature for inclusion in the Release Note blog article for the release that first includes this feature.__
+__Example article: http://wildfly.org/news/2018/08/30/WildFly14-Final-Released/.__
```

Comment:
Please change the link to https://www.wildfly.org/news/2024/01/25/WildFly31-Released/

That's a much more representative example of the way typical features are introduced.